oat-sa/extension-tao-itemqti
# Add checkbox for enable scrolling

Related to: https://oat-sa.atlassian.net/browse/NEX-2170
Rely to: https://github.com/oat-sa/tao-test-runner-qti-fe/pull/323

How to test:
- go to Item authoring tool
- Select or add textblock
- In properties should appear enable scrolling option for text block and shared stimulus

![image](https://user-images.githubusercontent.com/71755108/97469057-04e68f00-1957-11eb-9b90-242a51086b5f.png)

Also need:
